components/sensor/ppd42x.rst

```diff
@@ -0,0 +1,80 @@
+PPD42X Particulate Matter Sensor
+================================
+
+.. seo::
+    :description: Instructions for setting up PPD42X Particulate matter sensors
+    :image: ppd42x.jpg
+    :keywords: PPD42X
+
+The ``PPD42X`` sensor platform allows you to use PPD42X particulate matter sensors (`datasheet <https://www.researchgate.net/profile/Visa_Tasic/project/low-cost-sensor-platform-for-air-quality-monitoring/attachment/58332d5908ae91d0fe260283/AS:430912497295360@1479748953328/download/ShinyeiPPD42NS_TechDoc.pdf?context=ProjectUpdatesLog>`__)
+with ESPHome.
+
+.. figure:: images/ppd42-full.jpg
+    :align: center
+    :width: 50.0%
+
+    PPD42X Particulate Matter Sensor
+Wiring:
+Pin Function    Wire color (in picture above)
+1   Ground	                   Green
+2   P2 (output)                White
+3   5V                         Yellow
+4   P1 (output)                Black
+5   Threshold voltage for P2   Red
```

Comment:
This table won't show correctly. See waveshare_epaper as an example for these types of tables.